xbmc/pvr/recordings/PVRRecordingsPath.h

```diff
@@ -48,6 +50,8 @@ namespace PVR
     bool IsRecordingsRoot() const { return m_bRoot; }
     bool IsActive() const { return m_bActive; }
     bool IsDeleted() const { return !IsActive(); }
+    bool IsRadio() const { return m_bRadio; }
+    bool IsTV() const { return !IsRadio(); }
```

Comment:
Calling an inline method adds no overhead. I'm always  trying to encapsulate as much functionality as possible if it has no bad side effects, like unacceptable performance. Want to leave this as is .